packages/app-store/_utils/calendars/processExternalId.ts

```diff
@@ -0,0 +1,18 @@
+import type { DestinationCalendar } from "@prisma/client";
+
+import { metadata as OutlookMetadata } from "../../office365calendar";
+
+/**
+ * When inviting attendees to a calendar event, sometimes the external ID is only used for internal purposes
+ * Need to process the correct external ID for the calendar service
+ */
+const processExternalId = (destinationCalendar: DestinationCalendar) => {
```

Comment:
Do you think this function has the potential to grow in the number of "if" statements moving forward? i.e. We'll have to have different logic for a large number of different calendar apps. If so, we should move it off to its own file per calendar integration.

Since this is urgent to get fixed, that could be a follow-up refactor.

Reply:
The tests should written against main and not against this code first to verify all existing logic stays the same. Once verified, we then add tests to verify this change in logic